sphinx/src/contribution.rst

```diff
@@ -70,7 +70,7 @@ Since **Presto SQL:2016 Parser** is built by JavaCC from a Token based Grammar,
 
     -- <OVERLAPS:"OVERLAPS"> is a Token, recently defined in the Grammar
     -- Although it is not restricted by the SQL Standard and could be used for Column, Table and Alias names
-    -- Explicitly white-listing OVERLAPS by adding it to the  RelObjectNameWithoutValue() Production will allow for parsing the following statement
+    -- Explicitly white-listing OVERLAPS by adding it to the  NonReservedKeywords() Production will allow for parsing the following statement
```

Comment:
We try to use words like allow list and block list

Reply:
What a time to be alive! I will change it.

Reply:
What about ``white space characters`` though? :-))

Reply:
yeah so funny right - blank space?